kong/db/strategies/postgres/init.lua

```diff
@@ -902,7 +902,7 @@ function _M.new(connector, schema, errors)
         escaped = foreign_key_escaped,
       }
 
-    else
+    elseif not field.transient then
```

Comment:
Should we have another else branch to handle the case where `field.transient == true`?